### ISSUE TITLE
Sign In Fails to Launch Web Browser / Only shows blank screen

### ISSUE THREAD

**Issue by cod3monk3y** [alexa/ask-toolkit-for-vscode#124]
**Describe the bug**

In VS Code 1.56.0 and ASK Toolkit 2.5.0, clicking SKILLS MANAGEMENT > Sign In then clicking the Sign In button shows a blank window in VS Code, and does not launch a sign in page in browser (Firefox default). Downgrading to VS Code 1.53.x fixes this issue.

**To Reproduce**

1. Set Firefox as default browser (may/may not be necessary, but that's my setup)
2. Install VS Code 1.56.0
3. Install ASK Toolkit 2.5.0
4. Click **Sign In** under **SKILLS MANAGEMENT** of the **ALEXA SKILLS TOOLKIT** extension
5. In the **Sign In** tab that appears, click the **Sign in** button

**Extension Logs**

```
[info - 2021-05-07 19:48:00.933Z] Activating extension 
[error - 2021-05-07 19:54:51.168Z] Method not implemented.. 

[error - 2021-05-07 19:55:07.492Z] Method not implemented.. 

[error - 2021-05-07 19:57:18.342Z] Method not implemented.. 

[info - 2021-05-07 19:58:51.914Z] log level changed to verbose 
[debug - 2021-05-07 19:59:09.383Z] Calling method: ask.login 
[debug - 2021-05-07 19:59:09.383Z] Calling method: initialLogin.getHtmlForView 
[debug - 2021-05-07 19:59:09.383Z] Calling method: renderView, args:  [{"name":"initialLogin","js":true}]
[debug - 2021-05-07 19:59:23.728Z] Calling method: initialLogin.getHtmlForView 
[debug - 2021-05-07 19:59:36.889Z] Calling method: initialLogin.getHtmlForView 
[debug - 2021-05-07 19:59:37.464Z] Calling method: initialLogin.getHtmlForView 
```

**Expected behavior**

Tab should appear in Firefox for logging in to Amazon account.

**Actual behavior**

No tab appears in Firefox. Sign-in screen remains blank. No way to sign in to the extension.

**Screenshots**

![image](https://user-images.githubusercontent.com/1286619/117502413-54c09700-af34-11eb-83a6-f97af223abc8.png)

**Desktop (please complete the following information):**

- OS: Windows_NT x64 10.0.19042
- Visual Studio Code Version:  1.56.0
- Alexa Skills Toolkit Version: 2.5.0
- Git Version: git version 2.31.1.windows.1

**Additional context**

Default browser: Firefox 78.10.1esr (64-bit)

If you downgrade VS Code to 1.53.x, sign in will launch the browser tab as expected. Then upgrading back to 1.56.x again will re-introduce the  problem and it will not be possible to sign in. I did not investigate which version inbetween stopped working correctly.

**Comment by cod3monk3y** [comment]
Attempted installing 1.56.0 "System" from the direct link, rather than upgrading (https://code.visualstudio.com/updates/v1_56). Problem persists. Still unable to sign in with 1.56.0.

**Comment by carlos-algms** [comment]
I just tried with VSCode `v1.56.1` and I also receive the blank page.

**Comment by nikhilym** [comment]
Hey @cod3monk3y , @carlos-algms sorry for the late response. Thanks for getting this to our notice. You are right, the `sign in` button seems to be broken on the latest vscode version `v1.56.x`. I will check and raise a fix for this soon.

**Comment by rimolive** [comment]
This is the error that shows in VSCode Developer console when click on `Sign in` button: 

```
log.ts:296   ERR Unexpected token u in JSON at position 0: SyntaxError: Unexpected token u in JSON at position 0
	at JSON.parse (<anonymous>)
	at y (/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:79:5345)
	at s.$onMessage (/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:88:56403)
	at l._doInvokeHandler (/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:91:12836)
	at l._invokeHandler (/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:91:12520)
	at l._receiveRequest (/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:91:11187)
	at l._receiveOneMessage (/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:91:10172)
	at /usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:91:8074
	at fire (/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:57:1836)
	at S.fire (/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:65:15497)
	at /usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:106:29764
	at fire (/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:57:1836)
	at S.fire (/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:65:15497)
	at t._receiveMessage (/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:65:20755)
	at /usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:65:17641
	at fire (/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:57:1836)
	at acceptChunk (/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:65:12862)
	at /usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:65:12210
	at Socket.v (/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:106:13195)
	at Socket.emit (events.js:315:20)
	at addChunk (internal/streams/readable.js:309:12)
	at readableAddChunk (internal/streams/readable.js:284:9)
	at Socket.Readable.push (internal/streams/readable.js:223:10)
	at Pipe.onStreamRead (internal/stream_base_commons.js:188:23)
host.js:9 Uncaught TypeError: Cannot read property '1' of null
    at host.js:9
(anonymous) @ host.js:9
```

The issue still persists.

**Comment by nikhilym** [comment]
Hey @rimolive. Thanks for the error log. I saw you approved #127 . Do you see this issue happening even after the fix?

**Comment by rimolive** [comment]
No, the issue does not happen with the fix in #127.

**Comment by carlos-algms** [comment]
I'm seeing the blank screen when using the simulator for testing. 
Might be connected to this.
